src/tables.jl

```diff
@@ -24,6 +24,19 @@ getquery(r::Row) = getfield(r, :q)
 Tables.isrowtable(::Type{Query}) = true
 Tables.columnnames(q::Query) = q.names
 
+struct DBTable
+    name::String
+    schema::Union{Tables.Schema, Nothing}
+end
+
+DBTable(name::String) = DBTable(name, Tables.schema(""))
+
+DBTables = AbstractVector{DBTable}
```

Comment:
```suggestion
const DBTables = AbstractVector{DBTable}
```

why `AbstractVector` here instead of just `Vector`?

Reply:
No idea